tests/pytests/functional/utils/pkg/test_deb.py

```diff
@@ -0,0 +1,88 @@
+import salt.utils.pkg.deb
+from salt.utils.pkg.deb import SourceEntry
+
+
+def test__get_opts():
+    tests = [
+        {
+            "oneline": "deb [signed-by=/etc/apt/keyrings/example.key arch=amd64] https://example.com/pub/repos/apt xenial main",
+            "result": {
+                "signedby": {
+                    "full": "signed-by=/etc/apt/keyrings/example.key",
+                    "value": "/etc/apt/keyrings/example.key",
+                },
+                "arch": {"full": "arch=amd64", "value": ["amd64"]},
+            },
+        },
+        {
+            "oneline": "deb [arch=amd64 signed-by=/etc/apt/keyrings/example.key]  https://example.com/pub/repos/apt xenial main",
```

Comment:
Wondering if we should also test with arm64 too here, to ensure we don't miss something there.

Reply:
I can add that. but with what the function actually does it won't change anything. it doesn't do value checking on those values. or actually connect to any repos. it just pulls the variables out and apart.

Reply:
Saw that , just wanted an arm64 test in there too, to catch anyone in the future mucking up and making something Intel specific, so more a catch of future changes than yours which look fine.

Reply:
alright, added. to both this and SourceEntry.repo_line tests.